OktaLogger.podspec

```diff
@@ -17,7 +17,7 @@ Pod::Spec.new do |s|
   s.subspec "Complete" do |complete|
     complete.dependency 'OktaLogger/FileLogger'
     complete.dependency 'OktaLogger/FirebaseCrashlytics'
-    complete.ios.dependency 'OktaLogger/InstabugLogger'
+    complete.dependency 'OktaLogger/InstabugLogger'
```

Comment:
do we need this change?, we only support for iOS

Reply:
Without this change `pod lib lint` validation doesn't pass

Reply:
This might fail when you integrate in OV, please test it locally. macOS don't have Instabug support

Reply:
Double checked and everything is good